boring/src/sizing/test_R_evap.py

```diff
@@ -0,0 +1,49 @@
+#Author: K. Look  Created 10/27/20
+
+import os 
+cwd = os.path.dirname(os.path.realpath(__file__))
+chdir(cwd)
```

Comment:
you shouldn't need to do any path changes to run the tests, maybe look at the pattern in the other test files for guidance? I can also take a call whenever if you want to discuss